packages/legend-query/src/components/QueryBuilderExplorerPanel.tsx

```diff
@@ -244,14 +244,14 @@ const QueryBuilderExplorerPropertyDragLayer = observer(
 );
 
 const QueryBuilderExplorerContextMenu = observer(
-  (
-    props: {
+  forwardRef<
+    HTMLDivElement,
+    {
       queryBuilderState: QueryBuilderState;
       openNode: () => void;
       node: QueryBuilderExplorerTreeNodeData;
-    },
-    ref: React.Ref<HTMLDivElement>,
-  ) => {
+    }
+  >(function QueryBuilderExplorerContextMenu(props, ref) {
```

Comment:
Can we mentioned why we are doing this? Due to an upgrade in dependency?

Reply:
yes, this is warned by `mobx-react-lite`

```
[mobx-react-lite] `observer(fn, { forwardRef: true })` is depreacted, use `observer(React.forwardRef(fn))`
```